app/helpers.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+if (!function_exists('resource')) {
```

Comment:
Moved the helper here.

FYI This pattern of having a `helpers.php` file is used in laravel and lumen themselves. I've used it in platform-cloud-interface when porting a helper from laravel into lumen, I suspect we'll want to do that again in platform